backend/app/Repositories/Teams/EloquentTeamRepository.php

```diff
@@ -4,17 +4,15 @@
 
 namespace App\Repositories\Teams;
 
+use App\Entities\Website;
 use app\Repositories\Contracts\Teams\TeamRepository;
 use Illuminate\Support\Collection;
-use Illuminate\Support\Facades\DB;
 
 final class EloquentTeamRepository implements TeamRepository
 {
     public function getTeamMembers(int $websiteId): Collection
     {
-        return DB::table('users')
-            ->websites()
-            ->wherePivot('website_id', '=', $websiteId)
+        return Website::find($websiteId)->users()
             ->wherePivot('role', '=', 'member')
             ->select('name', 'email')
             ->get();
```

Comment:
instead of "select('name','email')->get()" you can use: get(['name','email']).
And select "member id"